deploy/templates/maintemplatev2/client/.gitignore

```diff
@@ -83,4 +83,13 @@ desktop/bind_windows_386.go
 desktop/windows.syso
 
 packer/build
-packer/packer
\ No newline at end of file
+packer/packer
+/lib/generated_plugin_registrant.dart
```

Comment:
maybe want to check this generated code in. 
We decided to always checking generated code so that CI does not need any tooling.
I know the system will compile with out this files because the i18n sysem runs off the json, but its worth sticking to always commit generated code globally for now.